engine/security/protect-socket.md

```diff
@@ -1,17 +1,41 @@
 ---
-description: How to setup and run Docker with HTTPS
-keywords: docker, docs, article, example, https, daemon, tls, ca,  certificate
+description: How to setup and run Docker with SSH or HTTPS
+keywords: docker, docs, article, example, ssh, https, daemon, tls, ca,  certificate
 redirect_from:
 - /engine/articles/https/
 - /articles/https/
+- /engine/https/
 title: Protect the Docker daemon socket
 ---
 
 By default, Docker runs through a non-networked UNIX socket. It can also
-optionally communicate using an HTTP socket.
+optionally communicate using SSH or a TLS (HTTPS) socket.
 
-If you need Docker to be reachable through the network in a safe manner, you can
-enable TLS by specifying the `tlsverify` flag and pointing Docker's
+## Use SSH to protect the Docker daemon socket
+
+The `docker` CLI can talk to a remote Docker daemon via SSH,
+by specifying `$DOCKER_HOST` to `ssh://<USERNAME>@<HOST>`.
```

Comment:
Perhaps would be good to;

- use a different username than `root` as example (to encourage setting up a user for docker; https://docs.docker.com/engine/install/linux-postinstall/)
- should we describe that `username` is optional (as it can be configured in `~/.ssh/config`) ?
- _slightly_ prefer (e.g.) `host1.example.com` to prevent confusing the above from being an e-mail address (`root@example.com` would be easily confused to be an e-mail address)
- I think we should describe using `docker context` as the "main" method to do this (assuming this is not for an "ad-hoc" situation)

Reply:
updated